addons/l10n_sa_edi/models/account_tax.py

```diff
@@ -0,0 +1,44 @@
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = 'account.tax'
+
+    l10n_sa_is_retention = fields.Boolean("Is Retention", default=False,
+                                          help="Determines whether or not a tax counts as a Withholding Tax")
+
+    l10n_sa_exemption_reason_code = fields.Selection(string="Exemption Reason Code",
```

Comment:
@mebe-odoo Then we should have it on tax template as well?

Reply:
@jco-odoo Done